storage/innobase/xtrabackup/doc/source/glossary.rst

```diff
@@ -4,73 +4,112 @@
+     contains the data of it (which is a standard Comma Separated Value file).
 
-         [mysqld]
-         innodb_file_per_table
+   .exp
+     Files with the ``.exp`` extension are created by |Percona XtraBackup| per
+     each |InnoDB| tablespace when the :option:`--export` option is
+     used on prepare. See :doc:`restoring individual tables <xtrabackup_bin/restoring_individual_tables>`.
 
-      or start the server with ``--innodb_file_per_table``.
+   .frm
+     For each table, the server will create a file with the ``.frm`` extension
+     containing the table definition (for all storage engines).
+
+   .ibd
+     On a multiple tablespace setup (:term:`innodb_file_per_table` enabled),
+     |MySQL| will store each newly created table on a file with a ``.ibd``
+     extension.
 
-   innodb_expand_import
-     This feature of |Percona Server| implements the ability to import
-     arbitrary :term:`.ibd` files exported using the |Percona XtraBackup|
-     :option:`--export` option.
+   .MRG
+     Each table using the :program:`MERGE` storage engine, besides of a
+     :term:`.frm` file, will have :term:`.MRG` file containing the names of the
+     |MyISAM| tables associated with it.
 
-     See the `the full documentation
-     <http://www.percona.com/doc/percona-server/5.5/management/innodb_expand_import.html>`_
-     for more information.
+   .MYD
+     Each |MyISAM| table has ``.MYD`` (MYData) file which contains the data on
+     it.
+
+   .MYI
+     Each |MyISAM| table has ``.MYI`` (MYIndex) file which contains the table's
+     indexes.
+   .opt
+     |MySQL| stores options of a database (like charset) in a file with a
+     :file:`.opt` extension in the database directory.
+
+   .par
+     Each partitioned table has .par file which contains metadata about the
+     partitions.
+
+   .TRG
+     File containing the Triggers associated to a table, for example
+     `:file:`mytable.TRG`. With the :term:`.TRN` file, they represent all the
+     Trigger definitions.
+
+   .TRN
+     File containing the Triggers' Names associated to a table, for example
+     `:file:`mytable.TRN`. With the :term:`.TRG` file, they represent all the
+     Trigger definitions.
+
+   backup
+      The process of copying data or tables to be stored in a different location. 
+
+   compression
+      The method that produces backups in a reduced size. 
+
+   configuration file
+      The file that contains the server startup options. 
+
+   crash
+      An unexpected shutdown which does not allow the normal server shutdown cleanup activities. 
+
+   crash recovery
+      The actions that occur when MySQL is restarted after a crash. 
+
+   data dictionary
+     The metadata for the tables, indexes, and table columns stored in the InnoDB system tablespace.
+  
+   datadir
+    The directory in which the database server stores its data files. Most Linux
+    distribution use `/var/lib/mysql` by default.
+
+   full backup
+
```

Comment:
full backup
A backup that contains the complete source data from an instance.

Reply:
Fixed.